view-transitions/mpa/index.js

```diff
@@ -0,0 +1,13 @@
+console.log("script loaded");
+
+window.addEventListener("pageswap", event => {
+    console.log("pageswap fired");
+    // event.viewTransition.skipTransition();
+});
+
+
+window.addEventListener("pagereveal", event => {
+    console.log("pagereveal fired");
+    // event.viewTransition.skipTransition();
```

Comment:
It’s not clear what they do and why they’re commented. Given the name, they probably let you skip the transition. But a small note should make it more straightforward :)

Reply:
Sorry, I didn't mean to include these. I'm going to remove the JS from the example altogether. I'll possibly include some JS demos separately as future updates.